.github/workflows/business-email-reminder-ci.yml

```diff
@@ -4,73 +4,18 @@ on:
   pull_request:
     types: [assigned, synchronize]
     paths:
-      - "jobs/email-reminder/**"
+      - "jobs/email_reminder/**"
+  workflow_dispatch:
 
 defaults:
   run:
     shell: bash
-    working-directory: ./jobs/email-reminder
+    working-directory: ./jobs/email_reminder
```

Comment:
is the renaming of the folder actually required for the jobs?

Reply:
![SmartSelect_20240201_092029_Chrome](https://github.com/bcgov/lear/assets/144159934/af2570bf-215e-4b5a-b050-ea3493f5b772)
I think so... Otherwise gonna have the linting error

Reply:
I would like to avoid renaming the folder for now just so we can still pick up changes from `main` when we do a sync.

Can you try doing what @leodube-aot does in his [PR](https://github.com/bcgov/lear/pull/2424) for the pylint config?  i.e. copy over the `[tool.pylint.main]` section